src/main/java/com/bootcamp/demo/dao/FirestoreDaoImpl.java

```diff
@@ -17,16 +16,15 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.Map;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 import java.util.concurrent.ExecutionException;
 
 import static java.lang.System.getProperty;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 @Repository
-public class FirestoreDaoImpl implements FirestoreDao{
+public class FirestoreDaoImpl implements FirestoreDao {
+    public static final String CARDS = "cards";
```

Comment:
You could rename this constant to `CARDS_COLLECTION` so it offers more information that its value.